docs/concepts/Package-Versioning.md

```diff
@@ -51,10 +55,12 @@ That said, package developers generally follow recognized naming conventions:
 - `-beta`: Beta release, typically one that is feature complete for the next planned release, but may contain known bugs.
 - `-rc`: Release candidate, typically a release that's potentially final (stable) unless significant bugs emerge.
 
+When resolving package references and multiple package versions differ only by suffix, NuGet chooses a version without a suffix first, then applies precedence to pre-release versions in reverse alphabetical order and treats dot notation numbers with numerical order. For example, the following versions would be chosen in the exact order shown:
```

Comment:
To be honest, I'm having a bit of a hard time grokking the first sentence.

Is it saying that NuGet sorts by descending SemVer 2 order, and then chooses the first (highest version)?

Reply:
Thanks. I think it's an improvement now.  Personally I'd prefer it to just say it sorts by descending SemVer2 order. I say we should get customers comfortable with understanding what SemVer2 is, whereas the change still might be interpreted as NuGet having some "custom" sort order.

But it's also fine as it is, we can wait to see if we get any customers asking questions where it appears they're confused by this.

Reply:
I agree with that.

I'll change it to make it explicit that it's semver, and not nuget order.